sfaira/data/dataloaders/loaders/d10_1038_nature22330/homosapiens_forebrain_2017_microwellseq_birey_001.yaml

```diff
@@ -0,0 +1,89 @@
+dataset_structure:
+    dataset_index: 1
+    sample_fns:
+dataset_wise:
+    author: "Birey, Fikri"
+    default_embedding: 
+    doi_preprint: 
+    doi_journal: "10.1038/nature22330"
+    download_url_data: "https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE93811&format=file"
+    download_url_meta: 
+    primary_data: True
+    year: 2017
+layers:
+    layer_counts: "X"
+    layer_processed: 
+    layer_spliced_counts: 
+    layer_spliced_processed: 
+    layer_unspliced_counts: 
+    layer_unspliced_processed: 
+    layer_velocity: 
+dataset_or_feature_wise:
+    feature_reference: 
+    feature_reference_var_key: 
+    feature_type: "rna"
+    feature_type_var_key: 
+dataset_or_observation_wise:
+    assay_sc: "microwell-seq"
+    assay_sc_obs_key: 
+    assay_differentiation: "Pasca, 2015" 
+    assay_differentiation_obs_key: 
+    assay_type_differentiation: "guided"
+    assay_type_differentiation_obs_key: 
+    bio_sample:
+    bio_sample_obs_key: 
+    cell_line: 
+    cell_line_obs_key: 
+    cell_type: 
+    cell_type_obs_key: 
+    development_stage: "14th week post-fertilization human stage"
+    development_stage_obs_key: 
+    disease: 
+    disease_obs_key: 
+    ethnicity: 
+    ethnicity_obs_key: 
+    gm:
+    gm_obs_key:
+    individual:
+    individual_obs_key: 
+    organ: "forebrain"
```

Comment:
if you specify "organ_obs_key", then "organ" must be empty. you either specify these terms per cell (ie. via obs key) or per dataset

Reply:
"sample" should also be listed as "organ_obs_key" as the different sample correspond to different organs.
just remoce the forebrain label for the whole dataset

Reply:
Alright.